packages/util/src/verkle.ts

```diff
@@ -38,7 +38,7 @@ export interface VerkleCrypto {
 /**
  * @dev Returns the 31-bytes verkle tree stem for a given address and tree index.
  * @dev Assumes that the verkle node width = 256
- * @param ffi The verkle ffi object from verkle-cryptography-wasm.
+ * @param ffi The verkle foreign function interface object from verkle-cryptography-wasm.
```

Comment:
Can we replace this with phrase with `{@link VerkleCrypto}` so it links directly to the verkle crypto interface. We should probably just change that ffi parameter name to `verkleCrypto` anyway since that's what we call it in most of the helpers.

Reply:
I renamed the `ffi` parameter to `verkleCrypto`. Fixed.